src/Core/src/Platform/iOS/LabelExtensions.cs

```diff
@@ -23,5 +24,31 @@ public static void UpdateTextColor(this UILabel nativeLabel, ILabel label)
 				nativeLabel.TextColor = textColor.ToNative(textColor);
 			}
 		}
+
+		public static void UpdateTextDecorations(this UILabel nativeLabel, ILabel label)
+		{
+			if (nativeLabel.AttributedText != null && !(nativeLabel.AttributedText?.Length > 0))
```

Comment:
How does `AttributedText` work? I see we always set `Text`, so will this be null, or does iOS create some wrapper for us?